File: Aac_h264_encode_decode/app/src/main/java/com/tao/aac_h264_encode_decode/LogUtils.java
/**
 * @ClassName: LogUtils
 * @Description:
 * @author taowei
 * @version V1.0
 * @Date
 */

package com.tao.aac_h264_encode_decode;

import android.util.Log;

public class LogUtils {
    private static final String TAG = "AacH264.";

    public static void d(String tag, String msg) {
        Log.d(TAG + tag,msg);
    }
}

File: SpeedBrowser/app/src/main/java/com/tw/speedbrowser/base/utils/LogUtils.kt
package com.tw.speedbrowser.base.utils

import android.util.Log

object LogUtils {
    private val TAG = "SpeedBs."

    fun d(tag: String, msg: String) {
        Log.d(TAG.plus(tag),msg)
    }

    fun e(tag: String, msg: String) {
        Log.e(TAG.plus(tag),msg)
    }
}